test/src/main/resources/DeploymentSpecifics.properties

```diff
@@ -38,6 +38,7 @@ jms.connectionfactory.fxf.activemq=jms/qcf-activemq
 jms.connectionfactory.fxf.mq=jms/qcf_fxf
 
 manageDatabase.serviceName=ibis4test-ManageDatabase
+manageFileSystem.serviceName=ibis4test-ManageFileSystem
```

Comment:
I recently moved the manageDatabase properties to the StageSpecifics property files. Could you pull from the master to fetch these changes and move the new `manageFileSystem` properties accordingly?